### PR TITLE
docs(artifacts): fix comment about total retry time

### PR DESCRIPTION
**Pull request by szymon-piechowicz-wandb**
# Description

Docs: https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html

# Test plan

n/a

### CODE REVIEW

File: wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py [hunk 1]
```diff
@@ -34,8 +34,8 @@
     from wandb.sdk.artifacts.artifact_manifest_entry import ArtifactManifestEntry
     from wandb.sdk.internal import progress
 
-# This makes the first sleep 1s, and then doubles it up to total times,
-# which makes for ~18 hours.
+# Sleep length: 0, 2, 4, 8, 16, 32, 64, 120, 120, 120, 120, 120, 120, 120, 120, 120
+# seconds, i.e. a total of 20min 6s.
```
Comment (moredatarequired) [inline wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py:38]: It also adds jitter, right? So could just be ~20 min.

File: wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py [hunk 2]
```diff
@@ -34,8 +34,8 @@
     from wandb.sdk.artifacts.artifact_manifest_entry import ArtifactManifestEntry
     from wandb.sdk.internal import progress
 
-# This makes the first sleep 1s, and then doubles it up to total times,
-# which makes for ~18 hours.
+# Sleep length: 0, 2, 4, 8, 16, 32, 64, 120, 120, 120, 120, 120, 120, 120, 120, 120
```
Comment (moredatarequired) [inline wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py:37]: To me at least it's actually a little clearer with words, e.g. "doubles (plus jitter) until it reaches the maximum of 2 minutes"

**Review by moredatarequired — APPROVED** [review]

File: wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py (hunk 1 above)
Comment (szymon-piechowicz-wandb) [reply wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py:38]: Jitter is 0 by default.

File: wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py (hunk 2 above)
Comment (szymon-piechowicz-wandb) [reply wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py:37]: It's a little more complicated than that and I didn't want to repeat the documentation.

**Pull request merged**